Services/PostService.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Blogger.Data;
+
+namespace Blogger.Services
+{
+    public class PostService
+    {
+        public BloggerContext Context;
+        public PostRepository PostRepo;
+
+        public PostService(BloggerContext bloggerContext)
+        {
+            Context = bloggerContext;
+            PostRepo = new PostRepository(Context);
+        }
+
+        public async Task<Post> Create(Post post)
+        {
+            if(post.Id != Guid.Empty)
```

Comment:
Why do we need that?

Reply:
`post.Id` will be set automatically by the system, so here I am checking whether this instance has its custom `Id` or not. Actually I don't know whether custom `Id` has any impact or not on creation.

Reply:
there is no issue when client side set `Id` by themselves. I removed this extra condition.